projects/packages/jetpack-mu-wpcom/src/features/wpcom-media/wpcom-media-url-upload.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Allows uploading media from URL in Media Library.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Appends the wpcom media URL upload form.
+ */
+function wpcom_media_url_upload() {
+	global $pagenow;
+
+	?>
+	<div id="wpcom-media-url-upload"></div>
+	<?php
+
+	$handle = jetpack_mu_wpcom_enqueue_assets( 'wpcom-media-url-upload', array( 'js', 'css' ) );
+
+	$data = wp_json_encode(
+		array(
+			'ajaxUrl'      => admin_url( 'admin-ajax.php' ),
+			'action'       => 'wpcom_media_url_upload',
+			'nonce'        => wp_create_nonce( 'wpcom_media_url_upload' ),
+			'isSiteEditor' => $pagenow === 'site-editor.php',
```

Comment:
Do we need to check whether the page is the post/page editor?

Reply:
I got the error on the post editor.

![image](https://github.com/user-attachments/assets/d753ca17-7bf7-42a5-a5a0-cdf6a09270c5)

Reply:
Woot, nice catch, thanks! I updated the PR 😄